src/tree.rs

```diff
@@ -699,6 +784,31 @@ impl<T> Default for Node<T> {
     }
 }
 
+#[cfg(test)]
+impl<T> std::fmt::Display for Node<T> {
```

Comment:
Is this any different than the `Debug` implementation?

Reply:
It is a bit more formated that with the `Debug` impl. The output looks like this : 
```
/ --> false
|---s --> false
|---|---d --> false
|---|---omething/ --> false
|---|---|---:a --> false
|---get/ --> false
|---doc/ --> false
|---|---rust --> false
|---bar --> false
|---|---/ --> false
|---a --> false
|---foo/ --> false
```

Reply:
I could improve it to make it look like this:
```rust
//! 9          \                1
//! 3          ├s               None
//! 2          |├earch\         2
//! 1          |└upport\        3
//! 2          ├blog\           4
//! 1          |    └:post      None
//! 1          |         └\     5
//! 2          ├about-us\       6
//! 1          |        └team\  7
//! 1          └contact\        8
```
Or remove it.